flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-starrocks/pom.xml

```diff
@@ -28,7 +28,7 @@ limitations under the License.
     <artifactId>flink-cdc-pipeline-connector-starrocks</artifactId>
 
     <properties>
-        <starrocks.connector.version>1.2.9_flink-${flink.major.version}</starrocks.connector.version>
+        <starrocks.connector.version>1.2.9_flink-1.18</starrocks.connector.version>
```

Comment:
It should be safe to upgrade to 1.2.10 with `flink-1.19` support.

(Some workarounds for `1.2.9_flink-1.18` could be removed, too.)